### DISCUSSION TITLE
Reorient faces coherentely

### DISCUSSION THREAD

**Discussion by wikyburns** [cnr-isti-vclab/PyMeshLab#169] [Q&A]
Hello guys, I'm trying to fix this 3D scan by orienting faces coherentely but the filter is doing nothing, any idea to fix that?

![image](https://user-images.githubusercontent.com/11525781/149460908-2ee15ef4-25d6-4c61-8886-03e2658e82ff.png)

STL : https://we.tl/t-TvjH8HiozT

**Comment by alemuntoni** [comment]
I think that this mesh will require first some other cleanups.
There is a discontinuity between the two portions of surface that have opposite orientations, and you can see this by executing the filter "select border":
![image](https://user-images.githubusercontent.com/6656168/149499315-2c7718c1-f677-4a31-b296-b81a6656c31c.png)
Therefore the resulting faces orientation is acceptable given that the two portions are not connected along that  border.
